### ISSUE TITLE
 about setting

### ISSUE THREAD

**Issue by yyj828** [qingshi9974/ICLR2024-FTIC#8]

**Comment by qingshi9974** [comment]
Your understanding is correct. In the training process, the first phase uses  ImageNet, with the hyperprior as the context model and ste_round as the quantizer, i.e., \hat{y} ={ste_round(y -means) + means. The GaussianConditional entropy encoder from CompressAI is used (with min_scale set to 0.11). Training lasts for about 2 million steps (with the learning rate decaying to  3e-5  during the last 200,000 steps), and the test loss converges to around 1.45 to 1.46 ( on Kodak testset).

The second phase uses either ImageNet or Flickr, with experiments showing that Flickr yields slightly better results. The context model and entropy encoder from the code are used (min_scale set to 0.01). After learning rate decay and fine-tuning on a 384x384 resolution dataset, the test loss converges to around 1.37. Fine-tuning on the 384x384 dataset reduces the test loss by approximately 0.01.
